train/uft/best_params.yml

```diff
@@ -1,7 +1,4 @@
-calc_eval_loss_interval: 200
-denoiser_adam_lr: 0.00012090676833404486
-denoiser_adversarial_loss_clip_max: 0.7253553053728639
-denoiser_adversarial_loss_clip_min: 0.42959290591546195
-denoiser_loss_b1: 0.654205478362656
-discriminator_adam_lr: 0.009627422411867004
-num_epochs: 11
+calc_eval_loss_interval: 381
```

Comment:
calc_eval_loss_interval isn't a tuneable parameter so can remove. but we don't need to run the hyper-parameter optimization again as it's not likely to change the result.

Reply:
ok updated, last time did not work but now can already